### PR TITLE
CSV dataset to test import opertation.update to update edges only [KAT-5432]

### PR DESCRIPTION
**Pull request by bhargavivadali**
CSV dataset to test import opertation.update to update edges only for KAT-2898 bug

### CODE REVIEW

**Review by e-mcginnis — APPROVED** [review]
LGTM

File: csv_datasets/random_graphs/small_graph/nodes_empty.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ID:ID(PERSON),NAME:STRING
```
Comment (gliga) [inline csv_datasets/random_graphs/small_graph/nodes_empty.txt:1]: this looks the same as nodes_empty_nodata.txt can we just use one (two times)?

**Review by gliga — APPROVED** [review]
Please make sure to update the name to be descriptive of what the graph has: graph_with_no_no_data or something like that.

Also see one comment about avoiding two files that look the same.

File: csv_datasets/random_graphs/small_graph/nodes_empty.txt (hunk 1 above)
Comment (bhargavivadali) [reply csv_datasets/random_graphs/small_graph/nodes_empty.txt]: updated the data in the file.

**Pull request merged**